src/test/java/com/mindee/parsing/common/OcrTest.java

```diff
@@ -31,6 +31,6 @@ void givenLicensePlatesV1_whenDeserialized_MustHaveAValidSummary() throws IOExce
       .readAllLines(Paths.get("src/test/resources/ocr/ocr.txt"));
     String expectedSummary = String.join(String.format("%n"), expectedLines);
 
-    Assertions.assertEquals(expectedSummary, prediction.getDocument().getOcr().toString());
+    Assertions.assertEquals(expectedSummary, prediction.getDocument().get().getOcr().toString());
```

Comment:
It would be better not to have to call the additional `get()` in order to access the Document object.

Reply:
Gotcha, thanks.